tools/topology/platform/intel/intel-generic-dmic-kwd.m4

```diff
@@ -42,7 +42,7 @@ ifdef(`DMIC_DAI_LINK_48k_NAME',`',define(DMIC_DAI_LINK_48k_NAME, `dmic01'))
 ifdef(`DMIC_DAI_LINK_16k_NAME',`',define(DMIC_DAI_LINK_16k_NAME, `dmic16k'))
 
 # DMICPROC is set by makefile, available type: passthrough/eq-iir-volume
-ifdef(`DMICPROC', , `define(DMICPROC, passthrough)')
+ifdef(`DMICPROC', , `define(DMICPROC, nr)')
```

Comment:
please dont modify existing pipelines, either add switches for your config or fork and make your own

Reply:
which one would be preferable? a switch or a new sof-tgl-max98373-rt5682-igonr.m4?

Reply:
i think copying is better in this case for the pipeline level m4, and add switches to top level m4

Reply:
Thanks! This looks better in this way.